internal/handlers/telegram/helpers.go

```diff
@@ -5,12 +5,25 @@ import (
 )
 
 /*
-GetUsername returns the name and username of a user. Since usernames are optional
+ResolveUserName does basic cleanup if a user does not have a username on Telegram.
+*/
+func ResolveUserName(u *tgbotapi.User) string {
```

Comment:
I guess it goes without saying that the ZWSP will only ever appear in IRC clients and not in Telegram clients. That is a correct assumption, right?

Reply:
Yes, this only appears on IRC, and does not affect Telegram.